README.md

```diff
@@ -45,3 +38,11 @@ at Function.Module._load (node:internal/modules/cjs/loader:828:14)
 ```
 then please try to downgrade to node 15 or node 14 (seems to be linked with
 https://github.com/electron-userland/electron-builder/issues/5668).
+
+## Legacy generators
+
+To generate legacy projects (not recommended), please run the following command:
+```shell
+npm install generator-symphony@1.7.7
```

Comment:
let's maybe detail that legacy projects are .net, node and legacy python/legacy java ones?

We would have to update https://docs.developers.symphony.com/developer-tools/developer-tools/symphony-bot-generator#quick-start
and remove https://docs.developers.symphony.com/building-bots-on-symphony/configuration/configure-your-bot-for-bdk  I would say

this is going to be outdated too https://docs.developers.symphony.com/building-bots-on-symphony/tutorials/sdk/build-an-interactive-bot-using-the-sdk